packages/core-js/internals/structured-clone.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable es/no-map -- safe */
+/* eslint-disable es/no-set -- safe */
+'use const';
+var isSymbol = require('./is-symbol');
+var toObject = require('./to-object');
+var getOwnPropertyNames = require('./object-get-own-property-names');
+var classof = require('./classof');
```

Comment:
Use paths from the top level of the package like for `../internals/classof` to avoid confusion and simplify refactoring in the future.